relayer/shannon/fullnode_lazy.go

```diff
@@ -178,60 +93,64 @@ type fullNode struct {
 	logger polylog.Logger
 }
 
-func (s *fullNode) LatestBlockHeight() (int64, error) {
-	return s.blockClient.LatestBlockHeight(context.Background())
+func (lfn *LazyFullNode) GetServiceApps(serviceID relayer.ServiceID) ([]apptypes.Application, error) {
+	allApps, err := lfn.getAllApps(context.TODO())
```

Comment:
Can you please add small 1-line comments above ever code section explaining wha/why we're doing? 

It'll make it easier to read.